frontend/components/siteList/siteList.js

```diff
@@ -74,7 +74,11 @@ export const SiteList = ({ storeState }) =>
     <AlertBanner {...storeState.alert} />
 
     {getSites(storeState.sites)}
-    {addWebsiteButton()}
+    <Link
```

Comment:
When I tried this locally, it didn't end up moving my scroll position up to the top.

Using a regular ole `a` tag did work, however: 

```html
<a href="#top">Return to top</a>
```